go/libraries/doltcore/doltdb/durable/index.go

```diff
@@ -147,6 +147,24 @@ type nomsIndex struct {
 
 var _ Index = nomsIndex{}
 
+func IterAllIndexes(
```

Comment:
Isn't this a method on either Schema or IndexSet?

Reply:
it is now!